librz/main/rz-find.c

```diff
@@ -143,7 +143,7 @@ static int hit(RzSearchKeyword *kw, void *user, ut64 addr) {
 		}
 	}
 	if (ro->exec_command) {
-		char *command = rz_str_newf("%s %s", ro->exec_command, ro->curfile);
+		char *command = rz_str_newf("%s ; grep -c . %s", ro->exec_command, ro->curfile);
```

Comment:
no.

Reply:
you could change the test file like this `CMDS=!rz-find -s "README" -E "echo" bins/arm~README~?`